Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -169,20 +169,19 @@ PlatformDeviceTableInitialize (
 
     SetDeviceTable (PltDeviceTable);
   }
+
+  // clear SmBus in use status as TCO timer reload causes it to be marked in use here
+  SmBusClearInUseStatus ();
```

Comment:
Does it impacts other ADL platforms?

Reply:
I am going to sanity test these changes on ADL-P or ADL-PS early next week. I will let you know.

Reply:
This does not impact other platforms. Confirmed on ADL-P today.